src/components/User/Trades/components/TradeFilters.vue

```diff
@@ -102,7 +103,13 @@ export default {
 </script>
 
 <style scoped>
+.app-list-filters {
```

Comment:
use bem

Reply:
This is a global class you can't override it.
It starts from 'app-list-filters' its should have been stopped you